RecoTracker/TkSeedGenerator/BuildFile.xml

```diff
@@ -8,7 +8,6 @@
 <use name="Geometry/CommonDetUnit"/>
 <use name="DataFormats/TrajectorySeed"/>
 <use name="DataFormats/TrackerRecHit2D"/>
-<use name="DataFormats/L1TrackTrigger"/>
```

Comment:
this better be moved to RecoTracker/TkSeedGenerator/plugins/BuildFile.xml due to direct use in 
RecoTracker/TkSeedGenerator/plugins/SeedGeneratorFromL1TTracksEDProducer.cc